test/validation/api/timer/timer.c

```diff
@@ -3266,17 +3242,52 @@ odp_suiteinfo_t timer_suites[] = {
 
 int main(int argc, char *argv[])
 {
+	int ret = 0;
+
 	/* parse common options: */
 	if (odp_cunit_parse_options(argc, argv))
 		return -1;
 
+	if (global_init())
+		return -1;
+
 	odp_cunit_register_global_init(timer_global_init);
 	odp_cunit_register_global_term(timer_global_term);
 
-	int ret = odp_cunit_register(timer_suites);
+	if (odp_cunit_register(timer_general_suites))
+		goto fail;
+
+	if (odp_cunit_run())
+		goto fail;
+
+	for (int i = ODP_CLOCK_SRC_0; i < ODP_CLOCK_NUM_SRC; i++) {
+		odp_timer_capability_t capa;
+
+		if (odp_timer_capability(i, &capa))
+			continue;
```

Comment:
Clock source 0 (=default) must be always supported. Is there a check for that, or should this CU_FAIL if capa claims that 0 is not supported?

Reply:
There's no specific test for that. timer_pool_sample_ticks fails if default source is not supported. I'll add a specific test so that it's clearer.